src/index.html

```diff
@@ -27,7 +33,7 @@
         </li>
       </ul>
       <span class="navbar-text">
-        Version
+        <a class="nav-link " id='leadLink' aria-current="page" href='#' onclick="ClickVersion()">Version</a>
```

Comment:
we are using MVVM, we should use the events as specified by the framework. 
https://angular.io/guide/event-binding